frontend/pages/api/workspace/getAWorkspace.ts

```diff
@@ -0,0 +1,31 @@
+import SecurityClient from '~/utilities/SecurityClient';
+
+interface Workspace {
+  __v: number;
+  _id: string;
+  name: string;
+  organization: string;
+  environments: Array<{ name: string; slug: string }>;
+}
+
+/**
+ * This route lets us get the workspaces of a certain user
+ * @returns
+ */
+const getAWorkspace = (workspaceID:string) => {
```

Comment:
I think this is a duplicate of GetProjectInfo.js
We kinda use workspaces and projects interchangeably which id pretty confusing